src/main/java/ganymedes01/etfuturum/recipes/ModRecipes.java

```diff
@@ -1554,18 +1551,10 @@ private static void removeFirstRecipeFor(Item item, int meta) {
 		}
 	}
 
-	private static void removeAllEFRRecipesFor(Block block) {
-		removeAllEFRRecipesFor(Item.getItemFromBlock(block));
-	}
-
 	private static void removeAllEFRRecipesFor(Block block, int meta) {
 		removeAllEFRRecipesFor(Item.getItemFromBlock(block), meta);
 	}
 
-	private static void removeAllEFRRecipesFor(Item item) {
-		removeAllEFRRecipesFor(item, OreDictionary.WILDCARD_VALUE);
-	}
-
```

Comment:
Unused ≠ Dead
Please refrain from removing utility code that may be used in the future.